src/components/Home.jsx

```diff
@@ -30,6 +47,7 @@ const Home = () => {
         return setSearchError('Please provide a search term');
       }
 
+      history.push(`/home/${searchWord}`);
```

Comment:
you could use global state to share searchWord across components instead passing it as a query string. Thus you won't need to parse query later, which is an additional effort. With that said this variant also works.

Reply:
I'm doing the whole query strings thing so that if the user refreshes the page/sends link to a friend, it will still show the same thing. :) I would've used context or lifted up the state otherwise

Reply:
Sounds fair!